### PR TITLE
fix: failed to update data

### PR DESCRIPTION
**Pull request by ebina4yaka**
### 实施细节
- 纠正了Axios中的导入语句。
- 执行 `yarn update-data` 

### 背景
最新的信息没有反映出来，所以我查看了 [arknights-toolbox-update](https://github.com/arkntools/arknights-toolbox-update) 的动作，发现之前的更新失败了。
https://github.com/arkntools/arknights-toolbox-update/actions/runs/3281076909/jobs/5402640990

### 错误的原因
这似乎是Axios v1.1.0以来一直存在的一个错误。一旦你删除了`.default`，它就会工作。
https://github.com/axios/axios/issues/5044

作为补充，如果你不使用Typescript类型推理，`require('axios')`就可以了。
https://axios-http.com/zh/docs/example
> 为了在CommonJS中使用 require（） 导入时获得TypeScript类型推断（智能感知/自动完成），请使用以下方法：

### CODE REVIEW

File: src/data/character.json [hunk 1]
```diff
@@ -75,8 +75,8 @@
     "star": 1,
     "recruitment": {
       "cn": 2,
-      "us": 2,
```
Comment (Zumurrud) [inline src/data/character.json:78]: Justice Knight is still in the US recruitment pool, so this line should be restored.

I think the problem with the update script is that the recruitment text says "Justice Knight", but her name elsewhere in the data is "'Justice Knight'" (note apostrophes).

Comment (Tsuk1ko) [reply src/data/character.json:78]: Yes, exactly as you said. I will fix this later.

Comment (ebina4yaka) [reply src/data/character.json:78]: Manually modified the json.

> I think the problem with the update script is that the recruitment text says "Justice Knight", but her name elsewhere in the data is "'Justice Knight'" (note apostrophes).

It seems that the data from which the data was obtained (ArknightsGameData) is causing some notational errors.

https://raw.githubusercontent.com/Kengxxiao/ArknightsGameData/master/en_US/gamedata/excel/gacha_table.json
https://raw.githubusercontent.com/Kengxxiao/ArknightsGameData/master/en_US/gamedata/excel/character_table.json

**Comment by Tsuk1ko** [comment]
Thanks for the PR :D

**Pull request merged**